pkg/entrypoint/entrypointer.go

```diff
@@ -18,13 +18,19 @@ package entrypoint
 
 import (
 	"fmt"
+	"io/ioutil"
 	"time"
 
 	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1alpha1"
 	"github.com/tektoncd/pipeline/pkg/logging"
 	"github.com/tektoncd/pipeline/pkg/termination"
 )
 
+const (
+	// DefaultResultPath is the path for task result
+	DefaultResultPath = "/tekton/results/"
```

Comment:
We probably need to create this directory on the file system somewhere so that its available for task containers. 

Im not sure if theres any place in the code where tekton performs the filesystem setup. I found this line https://github.com/tektoncd/pipeline/blob/c55d311ffc4b277f5a64a3787341d68d0288ff00/pkg/pod/workingdir_init.go#L71 that initializes the workspace. Im not sure if thats the right place for stuff under `/tekton` though.

Reply:
@bobcatfish @sbwsg any suggestions?

Reply:
I think the entrypoint should create that directory when it starts up if it doesn't exist already.  I don't see a need for that folder to be a shared volume (yet).